src/base/gps.h

```diff
@@ -55,6 +55,21 @@ class GPSTransform {
   std::vector<Eigen::Vector3d> XYZToEll(
       const std::vector<Eigen::Vector3d>& xyz) const;
 
+  std::vector<Eigen::Vector3d> EllToENU(
+      const std::vector<Eigen::Vector3d>& ell) const;
+
+  std::vector<Eigen::Vector3d> XYZToENU(const std::vector<Eigen::Vector3d>& xyz,
+                                        const double lat0,
+                                        const double lon0) const;
```

Comment:
Seems like this should be a private method?

Reply:
I don't really know here. It's not used yet anywhere else yet but might become handy for later use / applications? Also this will give me some troubles for the test functions ^^